src/Install/DBusRaucInstaller.cpp

```diff
@@ -447,6 +452,37 @@ namespace sua {
         return bundleVersion;
     }
 
+    TechCode DBusRaucInstaller::activate()
```

Comment:
For consistency activate() should call a private method callDBusRaucMark() (like install).

Reply:
Fixed